.github/workflows/test_url.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+for i in range(0,6):
+    command = "curl 'https://admin:admin@localhost:9200' -H 'Content-Type:application/json' --insecure -v"
+    result = os.system(command)
+    if result !=0:
+        os.system("sleep 30s")
+    else:
+        os.system("sleep 30s")
+        break  
```

Comment:
Can it be directly written in shell script as part of CI workflow?

Reply:
Yes, It can be added and corrected in commit [6eb89bc](https://github.com/opensearch-project/opensearch-api-specification/pull/35/commits/6eb89bc57d88f6a591f013932b3661ea58f9d480).